frontend/src/components/Seller/SellerListingForm.tsx

```diff
@@ -117,6 +139,19 @@ export default function SellerListingForm(props: ISellerListingFormProps) {
         required
         errors={errors}
       />
+      <Typography variant="h6"> Upload Cover Image </Typography>
+      {!isFileValidated && (
+        <Typography color={"red"}>
+          {" "}
+          Invalid File Type. Must be [png, jpg, jpeg]{" "}
+        </Typography>
+      )}
+      <input
```

Comment:
could you wrap this like they do in the docs for button uniformity https://mui.com/material-ui/react-button/#upload-button